### DISCUSSION TITLE
Participatory practices

### DISCUSSION THREAD

**Discussion by philhb** [UNMigration/HTCDS#15] [Ideas]
How might participatory practices be represented in the standards?

**Comment by OtiendeS** [comment]
I think where it would be easy to make the process participatory is in support planning. Already many organizations do participatory needs assessments and case planning offline. We can think of how these assessments can be incorporated and survivors can actually fill these assessments themselves.

**Comment by Anusha000000** [comment]
I am not sure if I have got the whole idea properly, but maybe participatory practices could be one way of making the standards more robust. In the development of Pathways, we had the ground-level expert review committee - so maybe involving survivors / potential victims from different countries to engage in these processes would be useful.

**Reply by philhb** [reply]
Are there any good books or websites you have which describe participatory best practices?

**Reply by Anusha000000** [reply]
The Institute of Development Studies website would have some good resources, though I am not sure if any are specific to participation in the context of case management. Will look out for some.

**Comment by Anusha000000** [comment]
Also, a good question to ask would be how long we would want the standards be work in progress - so periodic reflections of the standards to review the standards
